src/node_report_module.cc

```diff
@@ -84,6 +84,19 @@ static void SetCompact(const FunctionCallbackInfo<Value>& info) {
   per_process::cli_options->report_compact = compact;
 }
 
+static void GetDisableNetwork(const FunctionCallbackInfo<Value>& info) {
+  Mutex::ScopedLock lock(per_process::cli_options_mutex);
+  info.GetReturnValue().Set(per_process::cli_options->report_disable_network);
+}
+
+static void SetDisableNetwork(const FunctionCallbackInfo<Value>& info) {
+    Mutex::ScopedLock lock(per_process::cli_options_mutex);
```

Comment:
If this is a per-environment option there would be no need to use the locks. The `env->options()` can just be mutated safely.